ksqldb-rest-app/src/main/java/io/confluent/ksql/api/server/AuthHandlers.java

```diff
@@ -130,6 +150,11 @@ private static void selectHandler(
       context.next();
       return;
     }
+    if (skipPathsPattern.matcher(context.normalizedPath()).matches()) {
```

Comment:
Nit: Maybe put this first to match the previous order of checks in the plugin.  Shouldn't make any difference, but just to be safe.

Reply:
done